<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>820</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	background-color: rgb(222, 222, 220);
}</string>
  </property>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>1160</x>
     <y>20</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Open Sans ExtraBold</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: rgb(58, 134, 255);
	background-color: white;
	border: 2px solid rgb(58, 134, 255);
    border-radius: 15px;
}

QPushButton:hover:!pressed   {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:1 rgb(255, 189, 108), stop:0 rgb(255, 0, 0));
  color: white;
}

QPushButton:pressed   {
  background-color: rgba(255, 0, 0, 255);
  color: white;
}
</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1200</width>
     <height>800</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabWidget::pane { /* The tab widget frame */
    border-top: 2px solid rgb(222, 222, 220);
    position: absolute;
    top: 20 px;
}

QTabWidget::tab-bar {
    alignment: center;
}

/* Style the tab using the tab sub-control. Note that
    it reads QTabBar _not_ QTabWidget */
QTabBar::tab {
	color: rgb(58, 134, 255);
    background-color: white;
    border: 2px solid rgb(58, 134, 255);
    border-bottom-color: rgb(222, 222, 220);
	border-radius: 15px;
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    min-width: 15ex;
	top: 5 px;
    padding: 20px;

}

QTabBar::tab:selected, QTabBar::tab:hover {
background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0.857143, y2:0.857955, stop:1 rgba(10, 242, 251, 255), stop:0 rgba(224, 6, 159, 255));
}

QTabBar::tab:selected {
 	background-color: rgba(224, 6, 159, 255);
	color: white;
    border-color: #9B9B9B;
    border-bottom-color: #C2C7CB; /* same as pane color */
}</string>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="">
    <attribute name="title">
     <string>Menu</string>
    </attribute>
    <widget class="QLabel" name="drinks">
     <property name="geometry">
      <rect>
       <x>459</x>
       <y>2</y>
       <width>302</width>
       <height>120</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string>DRINKS HEADER</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="cocke_regular">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>160</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Coke (Regular)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="cocke_regular_count">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>162</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="pepsi_regular">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>210</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pepsi (Regular)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pepsi_regular_count">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>212</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="sprite_regular">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sprite (Regular)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="sprite_regular_count">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>262</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="string_regular">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>310</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>String (Regular)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="string_regular_count">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>312</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="coke_1_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>360</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Coke (1.5 LTR)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="coke_1_5_count">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>362</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="pepsi_1_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>410</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pepsi (1.5 LTR)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="pepsi_1_5_count">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>412</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sprite_1_5_count">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>462</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="sprite_1_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>460</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sprite (1.5 LTR)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="string_1_5_count">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>512</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="string_1_5">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>510</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>String (1.5 LTR)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="cups_count">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>562</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cups">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>560</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cups</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>119</y>
       <width>20</width>
       <height>521</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>640</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>750</x>
       <y>112</y>
       <width>20</width>
       <height>531</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>119</y>
       <width>20</width>
       <height>521</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="pepsi_1_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>410</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pepsi (1.5 LTR)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="sprite_1_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>460</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sprite (1.5 LTR)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="cocke_regular_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>160</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Coke (Regular)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="cups_count_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>562</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="drinks_2">
     <property name="geometry">
      <rect>
       <x>89</x>
       <y>2</y>
       <width>302</width>
       <height>120</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="text">
      <string>DRINKS HEADER</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="string_1_5_count_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>512</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="sprite_regular_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>260</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Sprite (Regular)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="pepsi_regular_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>210</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pepsi (Regular)</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="cocke_regular_count_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>162</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cups_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>560</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Cups</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="string_regular_count_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>312</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sprite_1_5_count_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>462</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="sprite_regular_count_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>262</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="coke_1_5_count_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>362</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>640</y>
       <width>301</width>
       <height>16</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QCheckBox" name="string_1_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>510</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>String (1.5 LTR)</string>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>112</y>
       <width>20</width>
       <height>531</height>
      </rect>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSpinBox" name="pepsi_regular_count_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>212</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="pepsi_1_5_count_2">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>412</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="coke_1_6">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>360</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Coke (1.5 LTR)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="string_regular_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>310</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>String (Regular)</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <attribute name="title">
     <string>Profile</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="">
    <attribute name="title">
     <string>Page</string>
    </attribute>
   </widget>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>exit_button</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
